models/db.py

```diff
@@ -776,6 +776,13 @@ def _count_custom_users_lambda(row):
                 Field("contest_id"),
                 Field("name"))
 
+db.define_table("problem_recommendations",
+                Field("user_id", "reference auth_user"),
+                Field("problem_id", "reference problem"),
+                Field("state", "integer"),
```

Comment:
Can you please document the states in the PR

Reply:
To confirm, add a comment in the code and not this PR, right?

Reply:
Yep, in the code